apps/server/src/modules/files-storage/service/files-storage.service.ts

```diff
@@ -143,6 +143,15 @@ export class FilesStorageService {
 		return promise;
 	}
 
+	private async sendToAntivirus(fileRecord: FileRecord): Promise<void> {
+		if (fileRecord.size > 25000000) {
```

Comment:
@SevenWaysDP Where do I find the correct env var for file security check size?

Reply:
It does not exist, please create it, next to "ENABLE_FILE_SECURITY_CHECK":